<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="studentText">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;SEBASTIAN RUD MADSEN&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;SEMAD17@STUDENT.SDU.DK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matrixvectorRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix-vector product</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="matrixmatrixRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix-matrix product</string>
    </property>
   </widget>
   <widget class="QLabel" name="calcLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the product to calculate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="matrixLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix dimensions (max of 9x9):</string>
    </property>
   </widget>
   <widget class="QLabel" name="vectorLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vector dimension (max of 9):</string>
    </property>
   </widget>
   <widget class="QLabel" name="secmatrixLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Second matrix dimensions (max of 9x9):</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstmatrixLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First matrix dimensions (max of 9x9):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="firstmatRowSpin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="firstmatColsSpin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="secmatRowsSpin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="secmatColsSpin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="firstmatRows">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rows:</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstmatCols">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secmatRows">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rows:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secmatCols">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstmatButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>841</width>
      <height>231</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Resultaterne af udregningerne er godt nok ikke korrekte, men det ser flot ud.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Hvis programmet crasher tilf√¶ldigt, kender jeg ikke noget til det.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Man kan godt indtaste doubles i en matrix, men det bliver lavet om til integer.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStandard">
    <property name="title">
     <string>Standard</string>
    </property>
   </widget>
   <addaction name="menuStandard"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>matrixvectorRadio</tabstop>
  <tabstop>matrixmatrixRadio</tabstop>
  <tabstop>firstmatRowSpin</tabstop>
  <tabstop>firstmatColsSpin</tabstop>
  <tabstop>firstmatButton</tabstop>
  <tabstop>secmatRowsSpin</tabstop>
  <tabstop>secmatColsSpin</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>studentText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
